### DISCUSSION TITLE
WPF plotter: any solution from a C# wrapper to a maptplotlib callable nested window?

### DISCUSSION THREAD

**Discussion by mrobbe** [CoolProp/CoolProp#2343] [General]
Is there a simple way (possibly with an example) about how to plot while using C# (WPF). 
In case it's too cumbersome has anyone implemented a wrapper using matplotlib and nesting the plot window within the xaml interface?
I have tried many different solution with scarce success and I am running out of ideas.
Thanks

**Comment by ibell** [comment]
I would be interested in any solution people find

**Comment by birdol** [comment]
Do you need to use WPF (C#) for plot chart, or do you need to use VBA (Excel) for plot chart? Or do you want to use VSTO WPF (C#) to draw charts in Excel?
